server/utils/schemas.ts

```diff
@@ -54,8 +44,10 @@ export const defaultValidatorJSON = getDefaults(validatorSchema) as ValidatorJSO
 
 export const mainQuerySchema = z.object({
   'payout-type': z.nativeEnum(PayoutType).optional(),
-  'only-active': z.literal('true').or(z.literal('false')).default('false').transform(v => v === 'true'),
   'only-known': z.literal('true').or(z.literal('false')).default('true').transform(v => v === 'true'),
   'with-identicons': z.literal('true').or(z.literal('false')).default('false').transform(v => v === 'true'),
   'force': z.literal('true').or(z.literal('false')).default('false').transform(v => v === 'true'),
+  'epoch-number': z.number().min(1).default(1),
```

Comment:
This will always fail since query params are strings, trying to use `epoch-number` as query param results in this:
```json
[ { "code": "invalid_type", "expected": "number", "received": "string", "path": [ "epoch-number" ], "message": "Expected number, received string" } ]
```

Example [here](https://9e0a0585.validators-api-mainnet.pages.dev/api/v1/validators?epoch-number=19)

Reply:
nice catch. forgot to `.transform(Number),`